trio/_core/_tests/test_ki.py

```diff
@@ -145,38 +146,38 @@ def protected_manager():
 
 
 @pytest.mark.skipif(async_generator is None, reason="async_generator not installed")
-async def test_async_generator_agen_protection():
+async def test_async_generator_agen_protection() -> None:
     @_core.enable_ki_protection
-    @async_generator
-    async def agen_protected1():
+    @async_generator  # type: ignore[misc] # untyped generator
```

Comment:
Just checking understanding, this is cause `async_generator` isn't typed?

Reply:
yeah, clarifying comment